src/ChunkSplitters.jl

```diff
@@ -9,13 +9,13 @@ export chunks, getchunk
 @compat public is_chunkable
 
 """
-    chunks(itr; n::Int, split::Symbol=:batch)
```

Comment:
Here may I would document the function as having two independent calls:
```julia
    chunks(itr; n::Int, split::Symbol=:batch)
    chunks(itr; size::Integer, split::Symbol=:batch)
```
maybe that makes it clearer that the two parameters are exclusive.

Reply:
I just wrote in the docstring that they are mutually exclusive, but I'm fine either way. I don't think I have ever seen what you're suggesting though.

Reply:
Me either. But to be fair, that is how I would like to see doc strings when a function has many methods, with a followup explanation of each method. But here it is not even the case. Just leave it as it is.